chainer/backends/cuda.py

```diff
@@ -1,20 +1,25 @@
 """Device, context and memory management on CuPy.
 
+.. note::
+   The package ``chainer.cuda`` has been renamed to
+   :mod:`chainer.backends.cuda` as of v4.0.0, but the previous module path
+   ``chainer.cuda`` is also available.
```

Comment:
Hmm, from that statement, users would think it's recommended to use `chainer.cuda`.

Reply:
Taking the fact that `cuda.to_gpu` etc. are one of the most frequently used API, no recommendation may confuse users, so I think it is better to clearly state that it will be kept for the foreseeable future.

Reply:
I think my concern was resolved by the description added to the upgrade guide.